pkg/cli/admin/rebootmachineconfigpool/command.go

```diff
@@ -0,0 +1,112 @@
+package rebootmachineconfigpool
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/dynamic"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+var (
+	rebootMachineConfigPoolLong = templates.LongDesc(`
+		Reboot the specified machine config pool by modifying an appropriate MachineConfig.
+
+		Does not wait for the reboot to complete, only initiates it.  This command will honor paused pools.
+		Degraded, failed, or otherwise not healthy nodes will not restart.
+
+		Experimental: This command is under active development and may change without notice.
+	`)
+
+	rebootMachineConfigPoolExample = templates.Examples(`
+		# Reboot all MachineConfigPools
+		oc adm clusteroperator reboot-machine-config-pool machineconfigpools --all
```

Comment:
nit: this is just `oc adm reboot-machine-config-pool`, no clusteroperator subcommand

Also, `--all` isn't a flag

Reply:
I promise to fix it later.